### PR TITLE
NEWS: add news for v0.28.0

### PR DESCRIPTION
**Pull request by ajeddeloh**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,16 @@
+22-Aug-2018 IGNITION v0.28.0
+
+  Features:
+
+    - Refactor blackbox tests to allow for testing with disks with 4k sectors
```
Comment (bgilbert) [inline NEWS]: s/allow for/allow/?
s/testing with/testing/?

File: NEWS [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+
+  Bug Fixes:
+
+    - Detect disks with 4k sectors correctly when scanning existing partitions
```
Comment (bgilbert) [inline NEWS]: `Correctly detect disks with 4k sectors when`

File: NEWS [hunk 3]
```diff
@@ -1,3 +1,16 @@
...
+    - Fix race between HTTP backoff tests
+    - Set the minimum config version to the actual minimum required in tests
```
Comment (bgilbert) [inline NEWS]: `Set the minimum config versions in tests to the actual minimum required`

File: NEWS [hunk 4]
```diff
@@ -1,3 +1,16 @@
...
+    - Relabel /root as well when relabeling with SELinux enabled
```
Comment (bgilbert) [inline NEWS]: `Relabel /root when SELinux relabeling is enabled`

**Comment by ajeddeloh** [comment]
Fixed

File: NEWS [hunk 5]
```diff
@@ -1,3 +1,16 @@
...
+
+  Features:
+
+    - Refactor blackbox tests to allow testing disks with 4k sectors
+
+  Bug Fixes:
+
+    - Correctly detect disks with 4k sectors when scanning existing partitions
+    - Fix race between HTTP backoff tests
+    - Set the minimum config versions in tests to the actual minimum required
```
Comment (bgilbert) [inline NEWS]: Trailing space.

**Pull request merged**